rust/flatbuffers/src/builder.rs

```diff
@@ -384,17 +370,19 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
     /// Speed-sensitive users may wish to reduce memory usage by creating the
     /// vector manually: use `start_vector`, `push`, and `end_vector`.
     #[inline]
-    pub fn create_vector_from_iter<T: Push + Copy>(
+    pub fn create_vector_from_iter<T: Push>(
```

Comment:
I added a blanked impl for Push for reference types, so that we can remove the Copy restriction here